docs/index.md

```diff
@@ -8,10 +8,13 @@ The GDK is composed of three layers:
 
 * **The GDK Core:** a performant, data-oriented integration with our cloud platform SpatialOS, based on the familiar Unity-native workflows.
 * **The GDK Feature Modules:** a library of solutions for hard or common networked game development problems, such as Character Movement and Shooting.
-* **The First Person Shooter (FPS) Starter Project:** a starter project for the GDK that enables you and your friends to experience the true scale of SpatialOS, providing a solid foundation for entirely new games.
+* **The GDK Starter Projects:**
+	* A [**First Person Shooter (FPS) Starter Project**]({{urlRoot}}/projects/fps/overview) that enables you and your friends to experience the true scale of SpatialOS, providing a solid foundation for entirely new games.
+	* A [**Blank Starter Project**]({{urlRoot}}/projects/blank/overview) containing the minimum GDK feature set you need to start developing games for SpatialOS.
 
 
 #### [Get started]({{urlRoot}}/content/get-started/get-started) with the SpatialOS GDK for Unity.
 
 
->This is the alpha release of the SpatialOS GDK for Unity. We invite projects to start using it but warn that all APIs are subject to change as we learn from feedback.
\ No newline at end of file
+
```

Comment:
Intentional?

Reply:
removed the extra newlines in both files now 👍